Makefile

```diff
@@ -20,6 +20,12 @@ PROXY_PATH=$(CMD_PATH)/$(PROXY)
 PROXY_CMD=./cmd/$(PROXY)
 PROXY_BIN=./bin/$(PROXY)
 
+# proxy service for Test Network
```

Comment:
I ask myself if we could avoid task duplication here. The proxy image is the same for carthagenet and mainnet right ?

Reply:
agreed